templates/default/deploy-scripts/src/deployer.rs

```diff
@@ -0,0 +1,394 @@
+use anyhow::{bail, Context, Error};
+use concordium_rust_sdk::smart_contracts::types::DEFAULT_INVOKE_ENERGY;
+use concordium_rust_sdk::types::hashes::TransactionHash;
+use concordium_rust_sdk::{
+    common::types::TransactionTime,
+    id::types::AccountAddress,
+    smart_contracts::common::ModuleReference,
+    types::{
+        queries::AccountNonceResponse,
+        smart_contracts::{ContractContext, InvokeContractResult, WasmModule},
+        transactions::{
+            self,
+            send::{deploy_module, init_contract, GivenEnergy},
+            InitContractPayload, UpdateContractPayload,
+        },
+        AccountTransactionEffects, BlockItemSummary, BlockItemSummaryDetails, ContractAddress,
+        Energy, TransactionType, WalletAccount,
+    },
+    v2::{self, BlockIdentifier},
+};
+use std::path::Path;
+use std::sync::Arc;
```

Comment:
Are you sure you formatted this? It looks like it was not formatted since there are multiple concordium_rust_sdk imports.

Reply:
yes, the files are formatted.

The formatter's default setting does not group the imports automatically.
https://github.com/Concordium/concordium-rust-smart-contracts/pull/324#pullrequestreview-1684302368

We could change the behavior of the formatter by e.g. adding a `rustfmt.toml` file and adding the below line in it.
```
imports_granularity = "Crate"
```

For now, I just run the formatter with the `imports_granularity = "Crate"` once to make sure we group similar imports together.